### PR TITLE
Tour feature

### PR DESCRIPTION
**Pull request by silvareal**
Tour Guide feature

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -299,9 +303,9 @@ class App extends Component {
                         <option value="tvseries"> TvSeries </option>
                     </select>
                     <div className="options__sub-details" >
-                    <button className="github-button" data-tour="my-first-step" onClick={this.startTour}> <WalkingIcon /> </button>
-                    <button className="switch-theme-btn" data-tour="my-eight-step" onClick={() => this.switchTheme(this.state.theme)}>{theme === 'dark'? <SunIcon /> : <MoonIcon />}</button>
-                    <a className="github-button" href="https://github.com/go-phie/gophie-web" data-tour="my-ninth-step"> <GitMark /> </a>
+                    <button className="actions-button tour-button" data-tour="my-first-step" title="Take A Tour" onClick={this.startTour}> <WalkingIcon /> </button>
+                    <button className="switch-theme-btn" data-tour="my-seventh-step" title="Change Theme" onClick={() => this.switchTheme(this.state.theme)}>{theme === 'dark'? <SunIcon /> : <MoonIcon />}</button>
```
Comment (deven96) [inline src/App.js:307]: data-tour="my-eight-step"

File: src/App.js [hunk 2]
```diff
@@ -299,9 +303,9 @@ class App extends Component {
...
+                    <a className="actions-button github-button" href="https://github.com/go-phie" data-tour="my-eight-step" title="Github Link" > <GitMark /> </a>
```
Comment (deven96) [inline src/App.js:308]: data-tour="my-ninth-step"

**Comment by deven96** [comment]
Please help me resolve those and reopen the PR

**Comment by silvareal** [comment]
Ok, resolved

**Pull request merged**